packages/mobile/src/geth/saga.ts

```diff
@@ -205,6 +213,8 @@ function* monitorAppState() {
 
 export function* gethSaga() {
   yield call(initGethSaga)
-  yield fork(monitorAppState)
-  yield fork(monitorGeth)
+  const gethRelatedSagas = yield all([fork(monitorAppState), fork(monitorGeth)])
+  yield take(Actions.CANCEL_GETH_SAGA)
+  yield cancel(gethRelatedSagas)
```

Comment:
Can you confirm this cancels the individual forks and not just the synthesized one created by `all()`? If monitorGeth's logs no longer show up after cancel that would confirm it :)

Reply:
Yes confirmed that monitorGeth logs go away! Agree that the redux saga cancel all documentation is unclear though (https://redux-saga.js.org/docs/advanced/TaskCancellation.html / outdated post https://github.com/redux-saga/redux-saga/issues/988#issuecomment-301064683)